### PR TITLE
de: update German files, add monero_guide

### PR DESCRIPTION
**Pull request by rodolfo912**

### CODE REVIEW

**Comment by erciccione** [comment]
would be nice to have a review of this translation. @rafficer @falcongoat ?

**Comment by Leza89** [comment]
I am a bit short on time currently.. maybe over the course of next week.

File: i18n/de/monero_guide.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+;black_list =
+```
+
+Diese Konfiguration öffnet einen Kovri-Listener auf dem standardmäßigen Testnet-P2P-Port von `monerod`. Lese als nächstes im [Benutzerhandbuch](https://github.com/monero-project/kovri-docs/blob/master/i18n/de/user_guide.md), wie man die Base32-Zieladresse erhält.
+
+Falls du bereits deinen Kovri-Router gestartet hast, führe `$ kill -HUP $(pgrep kovri)` aus, um den neuen Tunnel zu laden. Eine drastischere Möglichkeit besteht darin, einen harten Neustart durchzuführen, durch Stoppen und Starten von Kovri.
+
+## Kovri-Client-Tunnel
+
+Jetzt, wo der Server-Tunnel eingerichtet ist, starte die Einrichtung des Client-Tunnels durch Zeigen auf den Server-Tunnel.
```
[the remark this answers is not in the document]
Comment (rodolfo912) [reply i18n/de/monero_guide.md]: Yes, sounds better. Changed.

File: i18n/de/user_guide.md [hunk 2]
```diff
@@ -11,16 +11,16 @@ Hinweise:
 - Wenn du keinen Zugang zu deinem NAT hast, verwende die Laufzeitoption `--enable-upnp` oder schalte die Option in `kovri.conf` ein.
 - **Teile deine Portnummer mit niemandem, da dies deine Anonymität beeinträchtigen wird!**
 
-## Schritt 2. (Empfohlen) Betriebssicherheit (Opsec)
+## Schritt 2. (Empfohlen) Operative Sicherheit (Opsec)
 
-- Erwäge, einen festgelegten `kovri`-Benutzer zu erstellen und führe kovri nur mit diesem Benutzer aus
+- Erwäge, einen designierten `kovri`-Benutzer zu erstellen und führe kovri nur mit diesem Benutzer aus
```
[the remark this answers is not in the document]
Comment (rodolfo912) [reply i18n/de/user_guide.md]: I like "eigenen" in this context. Changed.

File: i18n/de/user_guide.md [hunk 3]
```diff
@@ -41,18 +41,19 @@ Für vollständige Optionen mit Details:
 
 ## Schritt 4. (Optional) Tunnel einrichten
 
-Kurz gesagt, *Client-Tunnel* sind Tunnel, über die du Verbindungen zu anderen Diensten herstellst, und *Server-Tunnel* werden verwendet, wenn du Dienste hostest (und andere Personen sich mit deinem Dienst verbinden).
+Kurz gesagt, *Client-Tunnel* sind Tunnel, über die du Verbindungen zu anderen Diensten herstellst, und *Server-Tunnel* werden verwendet, wenn du Dienste hostest, sodass sich andere Personen mit deinem Dienst (Website, SSH, etc.) verbinden können.
```
[the remark this answers is not in the document]
Comment (rodolfo912) [reply i18n/de/user_guide.md]: Changed.

File: i18n/de/user_guide.md [hunk 4]
```diff
@@ -41,18 +41,19 @@ Für vollständige Optionen mit Details:
...
 
 Du hast standardmäßig Client-Tunnel für IRC (Irc2P) und E-Mail (i2pmail) eingerichtet. Zum Hinzufügen/Entfernen von Client-Tunneln, siehe `tunnels.conf`.
 
-Beim Erstellen von Server-Tunneln musst du *persistente private Schlüssel* erzeugen. Entferne dafür das Kommentarzeichen bei bzw. schreibe `keys = your-keys.dat` und ersetze `your-keys` mit einem geeigneten Namen. **Teile deine private `.dat`-Datei mit niemandem und stelle sicher, dass du eine Sicherungskopie erstellst!**
+Beim Erstellen von Server-Tunneln musst du *persistente private Schlüssel* erzeugen, die für dein Ziel verwendet werden. Entferne dafür das Kommentarzeichen bei bzw. schreibe `keys = your-keys.dat` und ersetze `your-keys` mit einem geeigneten Namen. **Teile deine private `.dat`-Datei mit niemandem (hier besteht eine einzige Ausnahme und zwar, falls du Multihoming einsetzen möchtest) und stelle sicher, dass du eine Sicherungskopie der Datei erstellst!**
```
[the remark this answers is not in the document]
Comment (rodolfo912) [reply i18n/de/user_guide.md]: Changed.

File: i18n/de/user_guide.md [hunk 5]
```diff
@@ -41,18 +41,19 @@ Für vollständige Optionen mit Details:
...
 
-Nach der Einrichtung wird deine [Base32-Adresse](https://getmonero.org/resources/moneropedia/base32-address) in deinem Log angezeigt, nachdem du kovri gestartet hast. Du findest die Adresse auch in einer Textdatei zusammen mit der privaten Schlüsseldatei in deinem kovri-Datenpfad im `client/keys`-Ordner. Die Adresse in dieser `.txt`-Textdatei kann gefahrlos weitergegeben werden, damit sich andere Personen mit deinem Dienst verbinden können.
+Nach der Einrichtung wird dein [Base32](https://getmonero.org/resources/moneropedia/base32-address)- und [Base64](https://getmonero.org/resources/moneropedia/base64-address)-codiertes Ziel in deinem Log angezeigt, nachdem du kovri gestartet hast. Du findest diese Codierungungen auch in einer Textdatei zusammen mit der `.dat`-Datei in deinem kovri-Datenpfad im `client/keys`-Ordner. Das codierte Ziel in den **Textdateien** `.dat.b32.txt` und `.dat.b64.txt`kann gefahrlos an andere weitergegeben werden, damit sie sich mit deinem Dienst verbinden können.
```
[the remark this answers is not in the document]
Comment (rodolfo912) [reply i18n/de/user_guide.md]: Fixed.

**Comment by rodolfo912** [comment]
@Leza89 Thanks for your review! I fixed the mistakes and implemented all of your good suggestions.

**Pull request merged**